src/xmldiffng/fmes2.py

```diff
@@ -0,0 +1,304 @@
+from xmldiff import fmes
+from xmldiff.fmes import FmesCorrector
+
+class FmesCorrectorRNG(FmesCorrector):
+    def __init__(self, formatter, rngdict=None, f=0.6, t=0.5): # f=0,59
+        rngjson = '/suse/fweiss/Documents/werkstudent/bachelorarbeit/xmldiffng/contrib/rng.json'
+        self.rngdict = json.load(open(rngjson, 'r'))
```

Comment:
An alternative idea: don't load the JSON in your constructor, but do it "outside" and pass the rngdict on the constructor